presto-spi/src/main/java/io/prestosql/spi/connector/ConnectorPageSourceProvider.java

```diff
@@ -26,19 +26,6 @@ default ConnectorPageSource createPageSource(
             ConnectorSplit split,
             ConnectorTableHandle table,
             List<ColumnHandle> columns)
-    {
-        return createPageSource(transaction, session, split, columns);
-    }
-
-    /**
-     * @param columns columns that should show up in the output page, in this order
-     */
-    @Deprecated
```

Comment:
when it was deprecated?

Reply:
IIRC we had one issue where TupleDomain was common for a table scan and we are copying it for all splits so we capture it in TableHandle and pass it to the worker so the size of split can be reduced

Reply:
https://github.com/prestosql/presto/commit/e2e6fd3326d7b15ac40f22c16c237c30428c9a85